x/slashing/keeper/keeper.go

```diff
@@ -94,64 +93,6 @@ func (k Keeper) deleteMissingBlocksForValidator(batch db.Batch, address sdk.Cons
 	batch.Delete(key)
```

Comment:
if this errs, could it create problems for the validator? Maybe some logging?
Also, I noticed that some erred keeper operations panic but this is invisible. i.e., 
https://github.com/e-money/em-ledger/blob/b893cdc2432e711264eae77358309f9934bc64ec/x/slashing/keeper/keeper.go#L73
https://github.com/e-money/em-ledger/blob/b893cdc2432e711264eae77358309f9934bc64ec/x/slashing/keeper/keeper.go#L84
https://github.com/e-money/em-ledger/blob/b893cdc2432e711264eae77358309f9934bc64ec/x/slashing/keeper/keeper.go#L99

Reply:
If the errors above occur, they'll very shortly after make the validator lose consensus from the network. Panicking at these places ensure that there is at least a signal about the root cause location.